SConstruct

```diff
@@ -65,6 +65,8 @@ elif (os.name=="nt"):
 		custom_tools=['mingw']
 
 env_base=Environment(tools=custom_tools);
+if 'TERM' in os.environ:
+	env_base['ENV']['TERM'] = os.environ['TERM']
```

Comment:
I see you mentioned earlier that it was sufficient to get colored output, but how does it work? My `$TERM` is `xterm`, but that's not the terminal I'm using. Is that sufficient to trigger a coloured output in a different terminal?